src/cascadia/TerminalApp/TerminalTab.h

```diff
@@ -53,6 +53,7 @@ namespace winrt::TerminalApp::implementation
         void ResizeContent(const winrt::Windows::Foundation::Size& newSize);
         void ResizePane(const winrt::Microsoft::Terminal::Settings::Model::ResizeDirection& direction);
         void NavigateFocus(const winrt::Microsoft::Terminal::Settings::Model::FocusDirection& direction);
+        bool FocusPane(const uint32_t& id);

```

Comment:
This isn't part of a .idl right? In that case there's no need to pass this primitive type by reference.

Reply:
```suggestion
        bool FocusPane(const uint32_t id);
```